### PR TITLE
Updated PD JS3 W1

### PR DESCRIPTION
**Pull request by esma-g**
## What does this change?

Module: JS3
Week(s): 1

## Checklist

- [x] I have read the [contributing guidelines](https://syllabus.codeyourfuture.io/contributing/overview)
- [x] I have checked my spelling and grammar with an [automated tool](https://www.grammarly.com/grammar-check)
- [x] I have previewed my changes to check the [markdown renders](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) as I intend
- [x] I have run my code to check it works
- [x] My changes follow our [Style Guide](https://syllabus.codeyourfuture.io/guides/code-style-guide)

## Description

Updated PD JS3 W1

## Who needs to know about this?

@kfklein15 @CodeYourFuture/syllabus-team

### CODE REVIEW

File: docs/js-core-3/week-1/lesson.md
Comment (illicitonion) [inline docs/js-core-3/week-1/lesson.md]:
```suggestion
**Exercise objective:** To role-play a difficult conversation.
```

File: docs/js-core-3/week-1/lesson.md
Comment (illicitonion) [inline docs/js-core-3/week-1/lesson.md:490]: Is this meaningfully different from 2? If so, it may be worth calling out the focus of this one?

File: docs/js-core-3/week-1/lesson.md
Comment (illicitonion) [inline docs/js-core-3/week-1/lesson.md]: Should this be its own bullet point?
```suggestion
6. You have been receiving positive feedback all year long, but you didn’t get a promotion. However your peer did. You want to talk to your manager about it.
```

File: docs/js-core-3/week-1/lesson.md
Comment (illicitonion) [inline docs/js-core-3/week-1/lesson.md:493]: I'm not sure what difficult I'm expected to do as part of this scenario - it's maybe worth including a prompt for what the interviewer's feedback was?

**Review by illicitonion — DISMISSED** [review]
Looks good - thanks!

File: docs/js-core-3/week-1/lesson.md
Comment (esma-g) [reply docs/js-core-3/week-1/lesson.md:490]: I think they are different because no.2 is about personal relationship dynamics ("You think that they are not doing their share of the work on purpose because they dislike you.")

File: docs/js-core-3/week-1/lesson.md
Comment (esma-g) [reply docs/js-core-3/week-1/lesson.md:493]: The interviewer didn't provide feedback. The task is to ask for that feedback.

**Comment by esma-g** [comment]
@illicitonion I have a question. If I merge this PR now without signing off and committing your suggestions, will I have ignored those suggestions?

If I sign them off now, your review will become stale and I'll need another review before being able to merge. Help please :)

File: docs/js-core-3/week-1/lesson.md
Comment (illicitonion) [reply docs/js-core-3/week-1/lesson.md:490]: Maybe we could reframe one of them slightly to make that a little more clear? e.g. maybe no 2 could be "You think that they're not doing tasks you're dependent on on purpose because they dislike you"?

In general I'd hope that my approach to both "Someone's not doing their share" and "Someone's not doing their share, you think because they dislike you" would be exactly the same (because pre-supposing the reason someone's not meeting your expectations is generally harmful), so framing it as a slightly different problem seems worthwhile?

File: docs/js-core-3/week-1/lesson.md
Comment (illicitonion) [reply docs/js-core-3/week-1/lesson.md:493]: I guess my question stands then - what's the difficult part of this which they're meant to learn from? "Could I get feedback on my interview?" itself doesn't seem like a very interesting question/conflict/problem to discuss?

**Comment by illicitonion** [comment]
> If I merge this PR now without signing off and committing your suggestions, will I have ignored those suggestions?

Yes, those suggestions would be ignored.

In general it's nice to fold in the feedback before merging, but if stuff is super urgent you can merge ignoring the feedback and follow up later.

> If I sign them off now, your review will become stale and I'll need another review before being able to merge. Help please :)

First thing - I'm very happy to re-review (and generally if the changes are relatively small, same day :)) so don't worry about that part at all! Would definitely rather we work quickly together to get things into a good shape than be scared of having to wait for a re-review.

In general I tend to configure repos so that new changes _don't_ cancel out already given approvals; I assume that's not the way the syllabus repo is configured for a reason (though I will follow up with folks about that), so for now I'd say: unless there's a lot of urgency, let's push some changes to fix up the suggestions and I can just re-review quickly :)

File: docs/js-core-3/week-1/lesson.md
Comment (esma-g) [reply docs/js-core-3/week-1/lesson.md:490]: I have amended the sentence with your suggestion.

File: docs/js-core-3/week-1/lesson.md
Comment (esma-g) [reply docs/js-core-3/week-1/lesson.md:493]: It may be difficult for some trainees/recent graduates to ask this question. They are coming back from interviews without any feedback. I understand that it is not necessarily a difficult workplace conversation but it may be difficult for some.

I guess this was @kfklein15's addition. Karen, what do you think? Shall we amend/remove this scenario?

**Comment by esma-g** [comment]
> > If I merge this PR now without signing off and committing your suggestions, will I have ignored those suggestions?
> 
> Yes, those suggestions would be ignored.
> 
> In general it's nice to fold in the feedback before merging, but if stuff is super urgent you can merge ignoring the feedback and follow up later.
> 
> > If I sign them off now, your review will become stale and I'll need another review before being able to merge. Help please :)
> 
> First thing - I'm very happy to re-review (and generally if the changes are relatively small, same day :)) so don't worry about that part at all! Would definitely rather we work quickly together to get things into a good shape than be scared of having to wait for a re-review.
> 
> In general I tend to configure repos so that new changes _don't_ cancel out already given approvals; I assume that's not the way the syllabus repo is configured for a reason (though I will follow up with folks about that), so for now I'd say: unless there's a lot of urgency, let's push some changes to fix up the suggestions and I can just re-review quickly :)

Understood, thanks!

File: docs/js-core-3/week-1/lesson.md
Comment (kfklein15) [reply docs/js-core-3/week-1/lesson.md:493]: Let's leave it out for now and later think about how this could be used differently, @esma-g

**Review by illicitonion — APPROVED** [review]

**Pull request merged**
